src/main/java/core/basesyntax/service/DeepCopy.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.FruitTransaction;
+import java.util.List;
+
+public interface DeepCopy {
```

Comment:
The interface name 'DeepCopy' is too generic. Consider renaming it to something more specific to the context, such as 'TransactionListDeepCopy'.